src/App.jsx

```diff
@@ -1,5 +1,10 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
+const SORT_FIELD_LENGTH = 'length';
+const DEFAULT_SORT_FIELD = '';
```

Comment:
The `DEFAULT_SORT_FIELD` is set to an empty string. This might lead to confusion or unintended behavior since the sorting logic checks if `sortField` is truthy. Consider setting a more explicit default value or handling the empty string case explicitly in the sorting logic.